src/actions/warehouseActions.js

```diff
@@ -1,9 +1,38 @@
 import ActionTypes from './actionTypes';
 import ActionUtil from './actionUtil';
+import {getRoaster} from './roasterActions';
 
 const WAREHOUSE_URL = 'https://warehouse.expresso.store/api';
 const ITEMS_URL = WAREHOUSE_URL + '/item';
 
+function receiveItem(payload) {
+    return {
+        type: ActionTypes.WAREHOUSE_ITEM,
+        payload
+    };
+}
+
 export function getAllItems(offset, limit) {
     return ActionUtil.handlePagedRequest(ActionTypes.WAREHOUSE_ITEMS, ITEMS_URL, 'GET', offset, limit);
 }
+
+export function getItem(id) {
+    return dispatch => {
```

Comment:
The only reason I couldn't do that (I tried) was because I needed to call `dispatch(getRoaster(payload.data.roasterId));`, but I think I may be wrong in how I'm chaining dispatches.  Maybe this can be put in a util and we do a `.then(dispatch(getRoaster(...)))` in the container?